zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -334,6 +351,15 @@ def get_user_mentions(token: str) -> Tuple[str, int]:
         token = re.sub(slack_usermention_regex, mention, token)
         return mention, user_id
 
+    # Map italic, bold and strikethrough markdown
```

Comment:
This comment is better formatted as a docstring like the documentation in `channel_message_to_zerver_message`.